.circleci/config.yml

```diff
@@ -90,12 +92,24 @@ test_containers:
   - &memcached_port 11211
   - &container_agent
     image: datadog/agent
+    name: ddagent
     environment:
       - DD_APM_ENABLED=true
       - DD_BIND_HOST=0.0.0.0
       - DD_API_KEY=00000000000000000000000000000000
       - DD_HOSTNAME=dd-trace-rb-ci
   - &agent_port 8126
+  - &container_testagent
+    image: williamconti549/dd-apm-test-agent:pool-failures
```

Comment:
Will the image be replaced with the official one once we are happy with the implementation? https://github.com/orgs/DataDog/packages?repo_name=dd-apm-test-agent

Reply:
Yes it will, just waiting on a PR to be approved in the Test-Agent repository (which this image was produced from). I will switch the images once that PR is merged and released in the Test-Agent